src/dotnet/commands/dotnet-cache/LocalizableStrings.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace Microsoft.DotNet.Tools.Cache
+{
+    internal class LocalizableStrings
+    {
+        public const string AppFullName = ".NET Cache";
+
+        public const string AppDescription = "Cahes Assemblies specified for the .NET Platform, By Default these will be optimized for the target runtime and framework";
```

Comment:
Missing . at the end of the sentence. Also, why capital B after the comma: , By default?